pkg/sql/colflow/vectorized_flow.go

```diff
@@ -273,7 +274,10 @@ func (f *vectorizedFlow) GetPath(ctx context.Context) string {
 	// The directory name is the flow's ID.
 	tempDirName := f.GetID().String()
 	f.tempStorage.path = filepath.Join(f.Cfg.TempStoragePath, tempDirName)
-	log.VEventf(ctx, 1, "flow %s spilled to disk, stack trace: %s", f.ID, util.GetSmallTrace(2))
+	log.VEventf(
+		ctx, 1, "flow %s spilled to disk, stack trace: %s",
+		redact.SafeString(f.ID.String()), redact.SafeString(util.GetSmallTrace(2)),
```

Comment:
Don't call String on ID. It's already handled.

I think we already have a redactable variant of the small trace somewhere. If not, you can create it. It's not the responsibility of callers to assert it's safe.